fs_storage/models/fs_storage.py

```diff
@@ -339,6 +347,21 @@ def _get_fs_options(self):
             return options
         return self._eval_options_from_env(self.json_options)
 
+    def _get_marker_file_name(self):
+        return ".odoo_fs_storage_%s.marker" % self.id
+
+    def _check_connection(self, fs=False):
+        if not fs:
```

Comment:
I would make this a model method and make `fs` a required arg.

Reply:
@simahawk we use 'self' to generate the marker file name, maybe better to keep it without being a model method?

Reply:
Yeah, ok, but this thing of falling back to self.__fs is not need IMO. Either we expect to pass it or we don't.
Not a blocker tho if you think it makes sense :wink:

Reply:
@simahawk I agree that it looks better, could you check now?